### PR TITLE
WIP - LGA-1818 - Add steps for clicking the start button on the start page

### PR DESCRIPTION
**Pull request by said-moj**
Add steps for clicking the start button on the start page
Add steps for selecting the Education category and then the Special Educational Needs category.
Add step so taken to the Legal Aid is available for this type of problem page

### CODE REVIEW

File: behave/features/steps/cla_in_scope.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    assert_header_on_page("Check if you can get legal aid", context)
+    start_button = context.helperfunc.find_by_id('start')
+    assert start_button is not None
+    assert start_button.text == "Start now"
+    start_button.click()
+
+@given(u'I am on the scope diagnosis page')
+def step_scope_page(context):
+    wait_until_page_is_loaded("/scope/diagnosis/", context)
+    assert context.helperfunc.get_current_path() == "/scope/diagnosis/"
```
Comment (said-moj) [inline behave/features/steps/cla_in_scope.py:29]: Do we need this additional assert of the path as it is done by `wait_until_page_is_loaded` call above.

Comment (jogilder) [reply behave/features/steps/cla_in_scope.py]: Resolved by commit 8fc4c2

File: behave/features/cla_in_scope.feature [hunk 2]
```diff
@@ -5,6 +5,7 @@ Background: Start page
 
 @cla-in-scope
 Scenario: Select the in scope special education needs category
-#  GIVEN THAT I am on the scope diagnosis page
-#  WHEN I select categories Education -> Special Education needs
-#  THEN I am taken to the Legal aid is available for this type of problem page that is located on /legal-aid-available
\ No newline at end of file
+    Given I am on the scope diagnosis page
+    When I select the category Education
+    And the category Special Educational needs
```
Comment (jogilder) [inline behave/features/cla_in_scope.feature:10]: Should this 'and' be 'Then'? Then I select the category Special Educational needs?

Comment (Pheonnexx) [reply behave/features/cla_in_scope.feature:10]: I would have gone with 'And then select the category Special Educational needs' or, 'And I select the category "Special Educational needs"' which would make the category a variable and make the line re useable.

**Review by Pheonnexx — APPROVED** [review]
left a comment, but its more of a consideration or thought.

File: behave/features/cla_in_scope.feature (hunk 2 above)
Comment (jogilder) [reply behave/features/cla_in_scope.feature:10]: Ok, will look at updating this.

Comment (jogilder) [reply behave/features/cla_in_scope.feature:10]: I had a look and the two methods need to work differently. One button uses href to click through and the other (special educational needs) goes somewhere completely different than the href so can't really reuse the step anyway.

**Pull request merged**